cargo-insta/src/cli.rs

```diff
@@ -151,6 +151,9 @@ pub struct TestCommand {
     /// Build artifacts in release mode, with optimizations
     #[structopt(long)]
     pub release: bool,
+    /// Build artifacts with the specified profile
+    #[structopt(long)]
+    pub profile: String,
```

Comment:
Now profiles are required, the following doesn't work anymore:

```sh
cargo insta test
```

You need this:

```sh
cargo insta test --profile dev -- ui
```

Was this intended? If not I can create the fix PR with

```diff
-pub profile: String,
+pub profile: Option<String>,
```

Reply:
Oh boy, it's worse than that. The logic is looking at `features` rather than `profile`. Feel free to send a fix for both issues, else I'll do so tomorrow.

Reply:
I'll let you go for it because I didn't detect the other issue you just did :V

Reply:
https://github.com/mitsuhiko/insta/pull/407